.buildkite/pipeline.yml

```
@@ -1,13 +1,12 @@
 publish-android-artifacts-docker-container: &publish-android-artifacts-docker-container
-  plugins:
-    - docker#v3.8.0:
```

See the comment on the other PR for explanations of how to get the syntax you really wanted here.

TL;DR:
```suggestion
x-common-params:
  &publish-android-artifacts-docker-container
  docker#v3.8.0:
```
Then at cal site use:
```yml
plugins:
  - *publish-android-artifacts-docker-container
```

---

Thanks @AliSoftware! I've addressed this in 5e565b7c6f68bfce22036f5cfe2d4f42125e557c and it's looking great now 🥳